### PR TITLE
Feature/babelize

### PR DESCRIPTION
**Pull request by meltedspork**
I am not sure whats going with tests but it appears be working on docker

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
-    it('responds with package version', function(){
-      return path.expect(200, {version: process.env.npm_package_version});
-    });
-
-    it('responds with json', function(){
-      return path.expect('Content-Type', /json/);
-    });
-  });
-});
+    let path
```
Comment (kratsg) [inline test/test.js]: you misnamed the variables here. `var api = supertest(server)` and `api.get('/verson')` doesn't exist.

**Review by kratsg — CHANGES_REQUESTED** [review]
Apart from that, what would've helped make this review easier to understand is moving the files first before updating them.

**Comment by meltedspork** [comment]
Crikey - I think i will edit the PR to another branch instead to `master`

File: test/test.js (hunk 1 above)
Comment (meltedspork) [reply test/test.js]: I am still trying to figure where was misnamed?

**Comment by meltedspork** [comment]
test is still failing - even with minimal changes, not sure where i misnamed?

**Comment by kratsg** [comment]
@meltedspork you didn't make the right changes. Look at where you have `let path` because `api` is not defined...

**Comment by meltedspork** [comment]
Turns out stuff in test.js was red herring. What was broken was my change in `./src/index.js` export change - while worked fine with running the api but it wasn't exportable for test.js for some reason - most likely require more configuration with babel or something.

**Review by kratsg — APPROVED** [review]

**Pull request merged**